### PR TITLE
Sei txs, blocks, tx count pipeline

### PR DESCRIPTION
**Pull request by Ryan-Loofy**

### CODE REVIEW

File: models/silver/silver__temp.sql
Comment (eric-laurello) [inline models/silver/silver__temp.sql:1]: do we need this model?

File: .github/workflows/dbt_run_streamline_realtime.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+          cache: "pip"
+
+      - name: install dependencies
+        run: |
+          pip install -r requirements.txt
```
Comment (eric-laurello) [inline .github/workflows/dbt_run_streamline_realtime.yml:41]: I don't have a requirements.txt in this repo yet. Do we need to add it?

File: models/silver/silver__temp.sql
Comment (Ryan-Loofy) [reply models/silver/silver__temp.sql:1]: Yup for now - to quickly remake UDFs - will delete in PROD shortly after this PR

File: .github/workflows/dbt_run_streamline_realtime.yml (hunk 1 above)
Comment (Ryan-Loofy) [reply .github/workflows/dbt_run_streamline_realtime.yml:41]: Yup - added it

File: models/silver/streamline/_max_block_by_date.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{{ config (
+    materialized = "ephemeral",
+    unique_key = "block_number",
+) }}
+
+WITH base AS (
+
+    SELECT
+        block_timestamp :: DATE AS block_date,
+        MAX(block_number) as block_number
```
[the remark this answers is not in the document]
Comment (Ryan-Loofy) [reply models/silver/streamline/_max_block_by_date.sql:10]: Updated

File: models/silver/streamline/_max_block_by_date.sql [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{{ config (
+    materialized = "ephemeral",
+    unique_key = "block_id",
+) }}
+
+WITH base AS (
+
+    SELECT
+        block_timestamp :: DATE AS block_date,
+        MAX(block_id) as block_id
```
Comment (eric-laurello) [inline models/silver/streamline/_max_block_by_date.sql:10]: you may want to keep this named block number or we'll just need to change the other ref

File: macros/streamline/models.sql [hunk 4]
```diff
@@ -0,0 +1,179 @@
+{% macro decode_logs_history(
+        start,
+        stop
+    ) %}
+    WITH look_back AS (
+        SELECT
+            block_number
```
Comment (eric-laurello) [inline macros/streamline/models.sql:7]: this ref

**Review by eric-laurello — APPROVED** [review]

**Pull request merged**